Formula/iota.rb

```diff
@@ -22,6 +22,9 @@ class Iota < Formula
         end
     end
 
+    # Return with error if no compatible architecture was found.
+    odie "Unsupported architecture (#{RUBY_PLATFORM}). Please use cargo install and build from source" if arch == ""
```

Comment:
Hmm not too sure about that, there has to be a brew variable that would be better to use

Reply:
Or rather, why can't you use `arch` itself?

Reply:
Hmm not sure. Looks like `Hardware::CPU.arch.to_s` is also often used

Reply:
```suggestion
   odie "Unsupported platform #{Hardware::CPU.arch.to_s}-#{OS.kernel_name}" if arch == ""
```

Would print: `Error: Unsupported platform x86_64-Darwin`

or we use OS.mac and OS.linux but that seems more annoying